Samplero/Samplero/Screen/ShopBasket/ShopBasketViewController.swift

```diff
@@ -109,6 +115,7 @@ final class ShopBasketViewController: BaseViewController, ViewModelBindableType
         let layout = UICollectionViewFlowLayout()
         layout.scrollDirection = .vertical
         layout.minimumLineSpacing = Size.cellSpacing
+        layout.sectionInset = .init(top: 24, left: 0, bottom: 24, right: 24)
```

Comment:
p1;
```suggestion
        layout.sectionInset = .init(top: 24, left: 0, bottom: 24, right: 0)
```
right에 값이 들어가있었네영